nixos/modules/services/misc/home-assistant.nix

```diff
@@ -188,9 +183,13 @@ in {
     };
 
     package = mkOption {
-      default = pkgs.home-assistant;
+      default = pkgs.home-assistant.overridePythonAttrs (oldAttrs: {
```

Comment:
This shouldn't use `overridePythonAttrs` because that breaks the `override` that occurs later.

Reply:
@mweinelt 
I agree with @dotlambda, overriding the default package with `overridePythonAttrs` already proved problematic here:
https://github.com/NixOS/nixpkgs/commit/f9bd8b1b7bda019a823e93a0ecb719e15ac620cb
I think we should use `overrideAttrs` instead, like you did initially:
https://github.com/NixOS/nixpkgs/commit/1050f1487b34270dd9e8f362f54a15f1f5e01d9f

Reply:
So is the idea to revert f9bd8b1b7bda019a823e93a0ecb719e15ac620cb and keep 1050f1487b34270dd9e8f362f54a15f1f5e01d9f?